components/analytics-solutions-common/org.wso2.analytics.solutions.common/widgets/DateTimeRangePicker/src/CustomTimeRangeSelector.jsx

```diff
@@ -31,7 +31,7 @@ export default class CustomTimeRangeSelector extends React.Component {
             invalidDateRange: false
         };
 
-        this.startTime = new Date();
+        this.startTime = Moment().subtract(1, 'days').toDate();
```

Comment:
we are using simple "moment" in another file. Better to have a single convention

Reply:
Fixed with https://github.com/wso2/analytics-solutions/pull/306